README.md

```diff
@@ -26,50 +26,109 @@ versions are welcome.
 ```
 
-## What it sets up
+Choose the additional packages when the prompts appear:
 
+```
+Do you want to install Web's dependencies? [y|N]
+Do you want to install iOS's dependencies? [y|N]
+Do you want to install Android's dependencies? [y|N]
+```
+
+## What it sets up
+### Default
 * [Bundler] for managing Ruby libraries
-* [Docker] for managing project dependencies
-* [hub] for interacting with the GitHub API
-* [Heroku Toolbelt] for interacting with the Heroku API
+* [Keybase] to have encrypted team communication
+* [1Password] the world’s most-loved password manager
+* [Slack] for general team communication
+* [Google Chrome] as the default browser
+* [Skitch] get your point across with fewer words using annotation, shapes and sketches
+* [Postman] a collaboration platform for API development
+* [iTerm2] a replacement for Terminal
+* [git] distributed revision control system
+* [Github CLI] GitHub’s official command line tool
 * [Homebrew] for managing operating system libraries
-* [ImageMagick] for cropping and resizing images
-* [Postgres] for storing relational data
 * [RVM] for managing versions of Ruby
 * [NVM] for managing versions of Node.JS
-* [Redis] for storing key-value data
-* [Ruby] stable for writing general-purpose code
-* [Tmux] for saving project state and switching between projects
+* [The Silver Searcher] a code-searching tool similar to ack, but faster
 * [Zsh] as your shell
 * [oh-my-zsh] to spice up your shell
-* [Google Chrome] as the default browser
-* [Slack] for general team communication
-* [Keybase] to have encrypted team communication
+* [VS Code] code editor
+* [Watchman] watch files and take action when they change
+* [gpg2] GNU Pretty Good Privacy (PGP) package
+* [libyaml] YAML Parser
+* [coreutils] GNU File, Shell, and Text utilities
 
 [Bundler]: http://bundler.io/
-[Foreman]: https://github.com/ddollar/foreman
-[hub]: http://hub.github.com/
-[Heroku Toolbelt]: https://toolbelt.heroku.com/
+[Keybase]: https://keybase.io/
+[1Password]: https://1password.com/
+[Slack]: https://www.slack.com/
+[Google Chrome]: https://www.google.com/chrome/
+[Skitch]: https://evernote.com/products/skitch
+[Postman]: https://www.postman.com/
+[iTerm2]: https://www.iterm2.com/
+[git]: https://git-scm.com
+[Github CLI]: https://github.com/cli/cli
 [Homebrew]: http://brew.sh/
-[ImageMagick]: http://www.imagemagick.org/
-[NVM]: https://github.com/creationix/nvm
 [RVM]: https://rvm.io/
-[Postgres]: http://www.postgresql.org/
-[Redis]: http://redis.io/
-[Ruby]: https://www.ruby-lang.org/en/
+[NVM]: https://github.com/creationix/nvm
 [The Silver Searcher]: https://github.com/ggreer/the_silver_searcher
-[Tmux]: http://tmux.sourceforge.net/
 [Zsh]: http://www.zsh.org/
 [oh-my-zsh]: http://ohmyz.sh/
-[Docker]: https://www.docker.com/
-[Google Chrome]: https://www.google.com/chrome/
-[Slack]: https://www.slack.com/
-[Keybase]: https://keybase.io/
+[VS Code]: https://code.visualstudio.com/
+[Watchman]: https://github.com/facebook/watchman
```

Comment:
I think for iOS developer, we don't need this tool, it should be removed from the default list